modules/apps/adaptive-media/adaptive-media-image-test/src/testIntegration/java/com/liferay/adaptive/media/image/service/impl/test/AMImageEntryLocalServiceTest.java

```diff
@@ -735,6 +735,8 @@ private void _unregisterAMImageCounter(
 		}
 	}
 
+	private static final int _WELCOME_SITE_INITIALIZER_IMAGES_COUNT = 5;
```

Comment:
Question?

There is any way to obtain the number of images _before_ the test and create this way the counter.

If we make this a constant, this will happen other times if they change again the images.

Reply:
It probably exist, I chose the fastest solution 😅  I will check for this improvement 👍